x-pack/plugins/security_solution/docs/testing/test_plans/detection_response/prebuilt_rules/upgrade_review_algorithms.md

```diff
@@ -288,3 +348,24 @@ Examples:
 | algorithm        | field_name | base_version | current_version        | target_version         | merged_version                 |
 | array of scalars | tags       | N/A          | ["one", "two", "four"] | ["one", "two", "five"] | ["one", "two", "four", "five"] |
 ```
+
+#### **Scenario: `-BC` - Rule field is a solvable `data_source` object**
```

Comment:
I hadn't realized we are still calling this scenario `-BC`. We should rename all instances of this to `-AB` to have consitency with the new secnarios in the `ThreeWayDiffOutcome` enum